lib/filter.js

```diff
@@ -3,7 +3,7 @@
 var semver = require("semver");
 
 function isGitDependency(info) {
-    return info.latest === "git";
+    return ["git", "exotic"].indexOf(info.latest) !== -1;
```

Comment:
Could you rename that function to `isExoticDependency` then? I feel like that name would be more appropriate.